test/test_utils.py

```diff
@@ -317,7 +317,7 @@ def test_conda_as_dep(config_fixture, mulled_test):
         one:
           meta.yaml: |
             package:
-              name: one
```

Comment:
Test failed because some other `one==0.1=0` package is being picked up from the local package cache, i.e., some other test is not cleaned up properly.
Obviously, we should fix that, but it's not integral to this PR and I'm not motivated enough to go on that goose chase.